docs/DeveloperGuide.md

```diff
@@ -281,25 +281,36 @@ We categorise our user stories into four main epics:
+| S4  | `HIGH`   | user                                       | categorise my students with tags           |                                                                        |
+| S5  | `HIGH`   | user                                       | keep track of parent contacts              | communicate directly with the parent                                   |
+| S6  | `HIGH`   | user                                       | add remarks to a student                   | keep track of supplementary information                                |
+| S7  | `MEDIUM` | user                                       | keep track of a student's school           |                                                                        |
+| S8  | `MEDIUM` | user                                       | keep track of a student's academic level   | be reminded of the content that is relevant to the student             |
+| S9  | `MEDIUM` | user                                       | keep track of a student's outstanding fees | be reminded of the fees I have yet to collect                          |
+| S10 | `LOW`    | user                                       | have an archive for removed students       | revisit the archived data if the need arises                           |
+| S11 | `LOW`    | user with many students stored in **TAB**  | sort students by name                      | locate a student easily                                                |
+| S12 | `LOW`    | user with many students stored in **TAB**  | sort students by tag                       | locate a student easily                                                |
+| L1  | `HIGH`   | user                                       | schedule recurring lessons for a student   | keep track of when the student's lessons are                           |
+| L2  | `HIGH`   | user                                       | delete a student's lesson                  | remove lessons that I no longer have                                   |
+| L3  | `MEDIUM` | user                                       | edit a student's lesson                    | update lesson information                                              |
+| L4  | `MEDIUM` | user                                       | add makeup lessons for a student           | differentiate between the two different lessons                        |
+| L5  | `MEDIUM` | user                                       | view the lessons I have for the week       | easily manage my schedule                                              |
+| L6  | `MEDIUM` | user                                       | view a calendar of my scheduled lessons    | plan ahead while managing my schedule                                  |
+| U1  | `HIGH`   | new user                                   | see usage instructions                     | refer to instructions when I forget how to use **TAB**                 |
```

Comment:
"forget how to use TAB" sounds a little strange since the classification is for "new user" not "forgetful user". What do you think of replacing this with "refer to instructions whenever I need guidance on how to use TAB"?

Reply:
That's a great suggestion, I'll replace it.